components/camview.py

```diff
@@ -246,3 +255,31 @@ def onMove(self, axis, amount):
                 self.b += amount
             elif axis == Axis.C:
                 self.c += amount
+
+    def translate(self, newx=0, newy=0, newz=0):
+        if not self.trans:
```

Comment:
Would you write some comments explaining what is going on in this function? Also, it is hard for me to follow the logic related to self.trans. What does self.trans represents? In line 165, it checks if self.trans is true, but I do not see any case that self.trans becomes true outside the function. Is it something to implement in the future? Can you share your idea about it?